### DISCUSSION TITLE
Need for help

### DISCUSSION THREAD

**Discussion by Codexshield** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#674] [Q&A]
Hello and good evening, I would need help to install everything or to have the google assistant on my pc windows 11, I can't do it with the current explanations, I ask you if possible to help me via discord, preferably a person speaking French would be good, but if a person speaks English, it's fine, I would adapt
Thanks you for help 
Discord :  CodexShield#2475

**Comment by im-coder-lg** [comment]
I messaged you on Discord, the username is imcoderlg, same profile picture too.

**Comment by im-coder-lg** [comment]
But I can't do calls, so I will list the steps out in a way you could understand and you can follow that.
